docs/tasks.md

```diff
@@ -502,17 +539,17 @@ is configurable using a flag of the Tekton controller. If the configured "nop"
 image contains the command that the sidecar was running before the sidecar
 was stopped then the sidecar will actually keep running, causing the TaskRun's
 Pod to remain running, and eventually causing the TaskRun to timeout rather
-then exit successfully. Issue https://github.com/tektoncd/pipeline/issues/1347
+then exit successfully. Issue <https://github.com/tektoncd/pipeline/issues/1347>
 has been created to track this bug.
 
 ### Variable Substitution
 
 `Tasks` support string replacement using values from:
 
-* [Inputs and Outputs](#input-and-output-substitution)
-  * [Array params](#variable-substitution-with-parameters-of-type-array)
-* [`workspaces`](#variable-substitution-with-workspaces)
-* [`volumes`](#variable-substitution-with-volumes)
+- [Inputs and Outputs](#input-and-output-substitution)
+- [Array params](#variable-substitution-with-parameters-of-type-array)
```

Comment:
I *think* this one is supposed to be nested.  Looking for the heading below it appears to be one level smaller than `Inputs and Outputs`

Reply:
I don't see what you mean there. My editor reports issues if '*' are used as being inconsistent with other unordered list from the same document.

Reply:
Adding indentation to fix this.